content/en/docs/concepts/storage/volumes.md

```diff
@@ -92,7 +92,7 @@ Kubernetes supports several types of Volumes:
    * [scaleIO](#scaleio)
    * [secret](#secret)
    * [storageos](#storageos)
-   * [vsphereVolume](#vspherevolume)
+   * [vsphereVolume](#vsphereVolume)
```

Comment:
the anchor link for all the other volume types are all lowercased.  It may be better to change vspherevolume link to be lowercase to align with the others

Reply:
Done!